packages/api-client/src/main/tcp/ReconnectingWebsocket.ts

```diff
@@ -39,7 +39,15 @@ export enum WEBSOCKET_STATE {
   CLOSED = 3,
 }
 
+export enum PingMessage {
+  PING = 'ping',
+  PONG = 'pong',
+}
+
 export class ReconnectingWebsocket {
+  private static readonly CONFIG = {
+    PING_INTERVAL: TimeUtil.TimeInMillis.SECOND * 20,
+  };
```

Comment:
Why not move this outside of the class? It's static and private.

Reply:
Because it's private to `ReconnectingWebsocket` more than to the file.